app/controllers/api/service_dialogs_controller.rb

```diff
@@ -23,6 +23,17 @@ def create_resource(_type, _id, data)
       raise BadRequestError, "Failed to create a new dialog - #{e}"
     end
 
+    def edit_resource(type, id, data)
+      service_dialog = resource_search(id, type, Dialog)
+      service_dialog.update!(data.except('content'))
```

Comment:
should the update! above be inside the begin below so we can catch errors properly ?